src/main/scala/io/iohk/ethereum/jsonrpc/JsonMethodsImplicits.scala

```diff
@@ -29,10 +30,24 @@ object JsonMethodsImplicits {
 
   implicit val web3_clientVersion = new JsonDecoder[ClientVersionRequest] with JsonEncoder[ClientVersionResponse] {
     override def decodeJson(params: Option[JArray]): Either[JsonRpcError, ClientVersionRequest] = Right(ClientVersionRequest())
-
     override def encodeJson(t: ClientVersionResponse): JValue = t.value
   }
 
+  implicit val net_version = new JsonDecoder[VersionRequest] with JsonEncoder[VersionResponse] {
+    override def decodeJson(params: Option[JArray]): Either[JsonRpcError, VersionRequest] = Right(VersionRequest())
+    override def encodeJson(t: VersionResponse): JValue = t.value
+  }
+
+  implicit val net_listening = new JsonDecoder[ListeningRequest] with JsonEncoder[ListeningResponse] {
+    override def decodeJson(params: Option[JArray]): Either[JsonRpcError, ListeningRequest] = Right(ListeningRequest())
+    override def encodeJson(t: ListeningResponse): JValue = t.value
+  }
+
+  implicit val net_peerCount = new JsonDecoder[PeerCountRequest] with JsonEncoder[PeerCountResponse] {
+    override def decodeJson(params: Option[JArray]): Either[JsonRpcError, PeerCountRequest] = Right(PeerCountRequest())
+    override def encodeJson(t: PeerCountResponse): JValue = encodeAsHex(t.value)
```

Comment:
Maybe we should also remove the leading zeros here as the returned value is a `Quantity`?